<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FUNcube Dongle Controller</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qthid.qrc">
    <normaloff>:/qthid/images/qthid.png</normaloff>:/qthid/images/qthid.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEditFreq">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="labelFreqHz">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hz</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonDown">
          <property name="toolTip">
           <string>Frequency down</string>
          </property>
          <property name="statusTip">
           <string>Move frequency down</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditStep">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Frequency step (Hz)</string>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Frequency up</string>
          </property>
          <property name="statusTip">
           <string>Move frequency up </string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelCorrection">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Corr</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBoxCorr">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Frequency correction</string>
          </property>
          <property name="statusTip">
           <string>Frequency correction</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>-500</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="labelCorrection_2">
          <property name="text">
           <string>ppm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxFreqCorr">
        <property name="title">
         <string>Correction</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="labelDCI">
           <property name="text">
            <string>DC I</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxDCI">
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000030000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelGain">
           <property name="text">
            <string>Gain</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxGain">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000030000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDCQ">
           <property name="text">
            <string>DC Q</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxDCQ">
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000030000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelPhase">
           <property name="text">
            <string>Phase</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxPhase">
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000030000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxFw">
        <property name="title">
         <string>Firmware</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonVerifyFirmware">
           <property name="toolTip">
            <string>Verify the firmware on the FCD</string>
           </property>
           <property name="statusTip">
            <string>Verify the firmware</string>
           </property>
           <property name="whatsThis">
            <string>Verify the firmware in the Funcube Dongle against a firmware on disk</string>
           </property>
           <property name="text">
            <string>Verify</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonUpdateFirmware">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Upload new firmware</string>
           </property>
           <property name="statusTip">
            <string>Upload new firmware</string>
           </property>
           <property name="whatsThis">
            <string>Click this buttin to upload new firmware to the Funcube Dongle</string>
           </property>
           <property name="text">
            <string>Upload</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonBLReset">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Switch to application mode</string>
           </property>
           <property name="statusTip">
            <string>Switch to application mode</string>
           </property>
           <property name="whatsThis">
            <string>This button switches the Funcube Dongle back to application mode</string>
           </property>
           <property name="text">
            <string>Application</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonAppReset">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Switch to bootloader mode</string>
           </property>
           <property name="statusTip">
            <string>Switch to bootloader mode</string>
           </property>
           <property name="whatsThis">
            <string>This button switches the Funcube Dongle to bootloader mode where you can upload new firmware</string>
           </property>
           <property name="text">
            <string>Bootloader</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelPLLlock">
        <property name="text">
         <string>PLL Locked</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxPLLLock">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelIFRSSI">
        <property name="text">
         <string>IF RSSI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBarIFRSSI">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>70</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%p</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelRfChain">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="qthid.qrc">:/qthid/images/rfchain.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLNAGain">
        <property name="text">
         <string>LNA gain</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelMixerGain">
        <property name="text">
         <string>Mixer gain</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelMixerFilter">
        <property name="text">
         <string>Mixer filter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelIFGain1">
        <property name="text">
         <string>IF gain 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelIFRCfilter">
        <property name="text">
         <string>IF RC filter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelIFGain2">
        <property name="text">
         <string>IF gain 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="labelIFGain3">
        <property name="text">
         <string>IF gain 3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IF gain 4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelFilter">
        <property name="text">
         <string>IF filter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QLabel" name="labelIFGain5">
        <property name="text">
         <string>IF gain 5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="labelIFGain6">
        <property name="text">
         <string>IF gain 6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBoxLNAGain"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxRfFilter"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBoxMixerGain"/>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="comboBoxMixerFilter"/>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBoxIFGain1"/>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="comboBoxIFRCFilter"/>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="comboBoxIFGain2"/>
      </item>
      <item row="1" column="7">
       <widget class="QComboBox" name="comboBoxIFGain3"/>
      </item>
      <item row="1" column="8">
       <widget class="QComboBox" name="comboBoxIFGain4"/>
      </item>
      <item row="1" column="9">
       <widget class="QComboBox" name="comboBoxIFFilter"/>
      </item>
      <item row="1" column="10">
       <widget class="QComboBox" name="comboBoxIFGain5"/>
      </item>
      <item row="1" column="11">
       <widget class="QComboBox" name="comboBoxIFGain6"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelRFFilter">
        <property name="text">
         <string>RF Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLabel" name="labelBand">
          <property name="text">
           <string>Band</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelBiasCurrent">
          <property name="text">
           <string>Bias current</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="labelIFgainmode">
          <property name="text">
           <string>IF gain mode</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBoxLNAEnhance"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxBand">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBoxBiasCurrent"/>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="comboBoxIFGainMode"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelLNAEnhance">
          <property name="text">
           <string>LNA enhance</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDefaults">
        <property name="toolTip">
         <string>Reset FCD to default settings</string>
        </property>
        <property name="statusTip">
         <string>Load defaults</string>
        </property>
        <property name="text">
         <string>Defaults</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="fcdStatusLine">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionFirmware"/>
    <addaction name="actionBalance"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionBalance"/>
   <addaction name="actionFirmware"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="qthid.qrc">
     <normaloff>:/qthid/images/info.png</normaloff>:/qthid/images/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Information about Qthid</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>Information about Qt</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="qthid.qrc">
     <normaloff>:/qthid/images/quit.png</normaloff>:/qthid/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="qthid.qrc">
     <normaloff>:/qthid/images/open.png</normaloff>:/qthid/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load FCD settings from file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qthid.qrc">
     <normaloff>:/qthid/images/save.png</normaloff>:/qthid/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save FCD settings to file</string>
   </property>
  </action>
  <action name="actionBalance">
   <property name="icon">
    <iconset resource="qthid.qrc">
     <normaloff>:/qthid/images/iq.png</normaloff>:/qthid/images/iq.png</iconset>
   </property>
   <property name="text">
    <string>IQ Balance</string>
   </property>
   <property name="toolTip">
    <string>Open I/Q calibration window</string>
   </property>
  </action>
  <action name="actionFirmware">
   <property name="icon">
    <iconset resource="qthid.qrc">
     <normaloff>:/qthid/images/fw.png</normaloff>:/qthid/images/fw.png</iconset>
   </property>
   <property name="text">
    <string>Firmware</string>
   </property>
   <property name="toolTip">
    <string>Open firmware tools</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="qthid.qrc">
     <normaloff>:/qthid/images/help.png</normaloff>:/qthid/images/help.png</iconset>
   </property>
   <property name="text">
    <string>What's This?</string>
   </property>
   <property name="toolTip">
    <string>Enter What's This?</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qthid.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
